service/service.go

```diff
@@ -28,13 +28,20 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 
 	log.Info(ctx, "using service configuration", log.Data{"config": cfg})
 
-	r := mux.NewRouter()
-	r.Use(otelmux.Middleware(cfg.OTServiceName))
-	r.Use(serviceList.Init.DoGetRequestMiddleware().GetMiddlewareFunction())
+	router := mux.NewRouter()
+	router.Use(serviceList.Init.DoGetRequestMiddleware().GetMiddlewareFunction())
```

Comment:
this has re-ordered `r.Use(...)` calls such that OTel is now after this line, which may be what you want, but I suspect not - please confirm